src/BarChart/BarChartVerticalMultiSeries.story.tsx

```diff
@@ -173,193 +133,127 @@ export const StackedCustomStyle = () => (
-              rx={rx}
-              ry={ry}
-              guide={guide}
-              gradient={gradientTop}
-              rangeLines={rangelines}
-            />,
-            <Bar
-              rx={rx}
-              ry={ry}
-              guide={guide}
-              gradient={gradientBottom}
-              rangeLines={rangelines}
-            />
-          ]}
-          type="stackedDiverging"
-          colorScheme={chroma
-            .scale(['ACB7C9', '418AD7'])
-            .colors(data[0].data.length)}
-        />
-      }
-      yAxis={
-        <LinearYAxis
-          roundDomains={true}
-          tickSeries={
-            <LinearYAxisTickSeries
-              line={null}
-              label={<LinearYAxisTickLabel padding={5} />}
-            />
-          }
-        />
-      }
-      xAxis={
-        <LinearXAxis
-          type="category"
-          position="center"
-          tickSeries={<LinearXAxisTickSeries line={null} label={null} />}
-        />
-      }
-    />
-  );
-};
-
-export const StackedNormalized = () => {
-  const rx = number('rx', 0);
-  const ry = number('ry', 0);
-  const height = number('Height', 350);
-  const width = number('Width', 350);
-  const rangeWidth = number('Rangeline', 3);
-  const hasGradient = boolean('Gradient', true);
-  const hasRangelines = boolean('Rangelines', true);
-  const color = select('Color Scheme', schemes, 'cybertron');
-  const data = object('Data', multiCategory);
-
-  const gradient = hasGradient ? (
-    <Gradient
-      stops={[
-        <GradientStop offset="5%" stopOpacity={0.1} key="start" />,
-        <GradientStop offset="90%" stopOpacity={0.7} key="stop" />
-      ]}
-    />
-  ) : null;
-
-  const rangelines = hasRangelines ? (
-    <RangeLines position="top" strokeWidth={rangeWidth} />
-  ) : null;
-
-  return (
-    <StackedNormalizedBarChart
-      width={width}
-      height={height}
-      data={data}
-      series={
-        <StackedNormalizedBarSeries
-          bar={
-            <Bar
-              key="stacked-normalized-bar"
-              rx={rx}
-              ry={ry}
-              gradient={gradient}
-              rangeLines={rangelines}
-            />
-          }
-          colorScheme={color}
-        />
-      }
-    />
-  );
-};
-
-export const Marimekko = () => {
-  const rx = number('rx', 0);
-  const ry = number('ry', 0);
-  const height = number('Height', 350);
-  const width = number('Width', 350);
-  const rangeWidth = number('Rangeline', 3);
-  const hasGradient = boolean('Gradient', true);
-  const hasRangelines = boolean('Rangelines', true);
-  const color = select('Color Scheme', schemes, 'cybertron');
-  const data = object('Data', multiCategory);
-
-  const gradient = hasGradient ? (
-    <Gradient
-      stops={[
-        <GradientStop offset="5%" stopOpacity={0.1} key="start" />,
-        <GradientStop offset="90%" stopOpacity={0.7} key="stop" />
-      ]}
-    />
-  ) : null;
+export const StackedDiverging = () => (
```

Comment:
Should we be defaulting to `gradient={true} gridlines={true}` I can either export another component with different options or add a toggle